spec/ParseQuery.hint.spec.js

```diff
@@ -410,7 +410,7 @@ describe_only_db('mongo')('Parse.Query hint', () => {
     expect(queryPlanner.winningPlan.queryPlan.inputStage.inputStage.keyPattern).toEqual({ _id: 1 });
   });
 
-  it_only_mongodb_version('>=5.2')('query aggregate with hint (rest)', async () => {
+  it_only_mongodb_version('>=5.2')('query aggregate with hint (rest) (Mongo >=5.2)', async () => {
```

Comment:
I think we could modify the `it_only_mongodb_version` to print the MongoDB version automatically? Otherwise there are 2 items to maintain, the argument and the string, which is difficult to maintain, especially when tests are copied / modified.

Reply:
That might work considering we have multiple storage adapters